docs/user/03-traces.md

```diff
@@ -0,0 +1,400 @@
+
+  To configure an "always-on" sampling, configure a sampling rate of 100%:
+
+  ```yaml
+  apiVersion: telemetry.istio.io/v1alpha1
+  kind: Telemetry
+  metadata:
+    name: tracing-default
+    namespace: istio-system
+  spec:
+    tracing:
+    - providers:
+      - name: "kyma-traces"
+        randomSamplingPercentage: 100.00
+  ```
+
+  </details>
+  <details>
+  <summary label="namespaces-workload-settings">
+  Configure single Namespaces or workloads
+  </summary>
+  Namespaces or workloads can selectively be configured with individual settings by placing further resources. If you don't want to report spans at all for a specific workload, activate the `disableSpanReporting` flag with the selector expression.
+
+```yaml
+apiVersion: telemetry.istio.io/v1alpha1
+kind: Telemetry
+metadata:
+  name: tracing-default
+  namespace: my-namespace
+spec:
+  selector:
+    matchLabels:
+      kubernetes.io/name: "my-app"
+  tracing:
+  - providers:
+    - name: "kyma-traces"
+    randomSamplingPercentage: 100.00
+```
+
+  </details>
+  <details>
+  <summary label="trace-context-only">
+  Collect trace context without spans
+  </summary>
+  
+  To enable the propagation of the [w3c-tracecontext](https://www.w3.org/TR/trace-context/) only, without reporting any spans (so the actual tracing feature is disabled), you must enable the `kyma-traces` provider with a sampling rate of 0. With this configuration, you get the relevant trace context into the [access logs](https://kyma-project.io/docs/kyma/latest/04-operation-guides/operations/obsv-03-enable-istio-access-logs/) without any active trace reporting.
+
+  ```yaml
+  apiVersion: telemetry.istio.io/v1alpha1
+  kind: Telemetry
+  metadata:
+    name: tracing-default
+    namespace: istio-system
+  spec:
+    tracing:
+    - providers:
+      - name: "kyma-traces"
+      randomSamplingPercentage: 0
+  ```
+
+  </details>
+</div>
+### Eventing
+The Kyma [Eventing](https://kyma-project.io/docs/kyma/latest/01-overview/main-areas/eventing/) component dispatches events from an in- or out-cluster backend to your workload. It leverages the [CloudEvents](https://cloudevents.io/) protocol, which natively supports the [W3C Trace Context](https://www.w3.org/TR/trace-context) propagation. That said, the Eventing component already propagates trace context properly but does not enrich a trace with more advanced span data.
```

Comment:
will we have to rephrase this to "Eventing module" once it has been modularized?

Same question applies to all the other occurrences of the word "component".